extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -487,6 +493,12 @@ export class AmpA4A extends AMP.BaseElement {
               creativeParts.creative) {
             this.creativeBody_ = creativeParts.creative;
           }
+          if (creativeParts && creativeParts.size
```

Comment:
ok, then could you split out checking for the existence of just creativeParts then before this, and eliminate it from the two ifs? Otherwise, every time that creativeParts is null, you check it twice in a row

```
if (!creativeParts) {
  return Promise.resolve();
}
```